haiku-games/bong/bong-1.4~git.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="BeOS version of Pong"
+DESCRIPTION="Bong is a FreeWare game for BeOS, based on the old arcade classic Pong"
+HOMEPAGE="https://github.com/HaikuArchives/Bong"
+COPYRIGHT="1997 by Gertjan van Ratingen"
+LICENSE="bong"
+REVISION="1"
+srcGitRev=906d822b79bdae9328fe743dffe2871f96a30e0e
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="bong-1.4~$srcGitRev.tar.gz"
+SOURCE_DIR="Bong-$srcGitRev"
+PATCHES="bong-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	bong = $portVersion
+	app:bong = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -i -e "/^#if 0$/,/^#endif$/ s/.*//;" bong.cpp
+}
+
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects MACHINE=BePC CC=g++ LD=g++
```

Comment:
Sorry, I had this change to submit but forgot to do so!
```diff
-	make $jobArgs OBJ_DIR=objects MACHINE=BePC CC=g++ LD=g++
+	make $jobArgs OBJ_DIR=objects
```
The app builds fine on x86_gcc2, and even runs well, except that the keyboard controls do not work :(